Automattic/wp-calypso
# Importer flow: Mark find_available_url to true if from migration flow

Related to #77473 

## Proposed Changes

* Make sure the site creation step on the migration plugin flow always finds available URL.

## Testing Instructions



* Start with `http://calypso.localhost:3000/setup/newsletter/intro`, go through steps and setup a newsletter
* Prepare a JN site with Move to WordPress.com plugin and click `Get started` CTA
* Click `create a new one` button on sitePicker step
* Make sure a new site is created

## Pre-merge Checklist
